apps/web/src/components/Menu/AddressBookMenu.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button, Divider, DrawerBody, DrawerContent, Heading, VStack } from "@chakra-ui/react";
+
+import { useColor } from "../../styles/useColor";
+import { DrawerBackButton } from "../BackButton";
+import { EmptyMessage } from "../EmptyMessage";
+import { ModalCloseButton } from "../ModalCloseButton";
+
+export const AddressBookMenu = () => {
+  const color = useColor();
+
+  return (
+    <DrawerContent>
+      <DrawerBackButton />
+      <ModalCloseButton />
+      <DrawerBody paddingTop="90px">
+        <Heading color={color("900")} fontSize="24px">
```

Comment:
it matches size xl. please use it and in other places too

Reply:
It doesn't match this size, but I added 2xl to match 24px. 
<img width="565" alt="image" src="https://github.com/user-attachments/assets/650f636c-94de-46be-9c4b-5e478b5bcecd">

Reply:
sorry, I meant 2xl. thanks